layouts/partials/app_and_api_protection/java/overview.html

```diff
@@ -0,0 +1,12 @@
+<h2>Overview</h2>
+
+<p>Datadog Application Security Management (ASM) provides App and API Protection (AAP) capabilities including:</p>
+<ul>
+  <li><strong>Application Security Monitoring</strong>: Real-time threat detection and protection against attacks like SQL injection, XSS, and more</li>
+  <li><strong>Software Composition Analysis (SCA)</strong>: Identification of vulnerable dependencies in your codebase</li>
+  <li><strong>Interactive Application Security Testing (IAST)</strong>: Runtime vulnerability detection during testing</li>
+</ul>
+
```

Comment:
ASM no longer exists - we replaced it with AAP. It doesn't encompass IAST not SCA, which are part of Code Security, and out of the scope of this documentation update. We'll work on update them later on.

```suggestion
```